solid-team.md

```diff
@@ -9,16 +9,16 @@ This document aims to identify the various roles within Solid, and the responsib
 
 ## Solid Manager
-The Solid Manager implements the governing vision of the Solid project. The Solid Manager is responsible for coordination and collaboration to develop the Solid specifications and the implementation of the Solid specifications as well as the Solid Advisory Board. The Solid Manager is responsible for moderating the W3C Solid Community Group calls. The Solid Manager is responsible for keeping the W3C Solid Community Group information up to date including Solid solutions and contributors as well as encouraging and supporting Solid Event organisers. The Solid Manager is responsible for coordinating Solid User testing. The Solid Manager has admin rights of all Solid repositories. 
+The Solid Manager implements the governing vision of Solid. The Solid Manager is responsible for coordination and collaboration to develop the Solid specifications and the implementation of the Solid specifications as well as the Solid Advisory Board. The Solid Manager is responsible for moderating the W3C Solid Community Group calls. The Solid Manager is responsible for keeping the Solid information up to date and is therefore responsible for keeping the properties of their Solid information repositories well-organised and communicating any changes clearly and publicly. The Solid Information Repository Manager has admin rights of the Solid information repositories and is expected to provide resolution and a defined route forward on any difference of opinion that may occur. The Solid Manager is responsible for encouraging and supporting Solid Event organisers as well as Solid Projects which involves the publication of the project name, project scope as well as allocation of the Project Manager and Project Contributors. The Solid Manager has admin rights of all Solid repositories. 
 
 ## Solid Specifications Repository Manager
 The Solid Specification Repository Manager is responsible for keeping the properties of their Solid specification repositories well-organised and communicating any changes clearly and publicly. The Solid Specification Repository Manager has admin rights of the Solid specification repositories and is expected to provide resolution and a defined route forward on any difference of opinion that may occur.
 
 ## Solid Experts 
-The Solid Experts are responsible for providing support to those looking to implement the Solid specification. 
+Solid Experts provide support and guidance to those looking to implement the Solid specification.They have admin rights on all Solid repositories, and are expected to participate in regular community group meetings, and maintain availability for project specific collaboration as needed.
```

Comment:
* `specification.They` -> `specification. They`
* `and are expected to participate in regular community group meetings, and maintain availability` -> `and are expected both to participate in regular community group meetings and to maintain availability`